Makefile

```diff
@@ -10,7 +10,7 @@ unit-test:
 
 .PHONY: test-snapshot-update
 test-snapshot-update:
-	$(run) pytest --cov-report term-missing --cov=textual tests/ -vv --snapshot-update
+	$(run) pytest --cov-report term-missing --cov=textual tests/ -vv --snapshot-update $(PYTEST_ARGS)
```

Comment:
Note to reviewer: I took the liberty to add this snippet, which let you do things like: `make PYTEST_ARGS="-k digit_display" test-snapshot-update` while developing.
Let me know if you prefer that I remove this.